Engine/DataFeeds/DataManager.cs

```diff
@@ -175,10 +183,18 @@ public int SubscriptionManagerCount()
             var result = (from subscriptionDataType in dataTypes
                 let dataType = subscriptionDataType.Item1
                 let tickType = subscriptionDataType.Item2
-                select new SubscriptionDataConfig(dataType, symbol, resolution, marketHoursDbEntry.DataTimeZone,
-                    exchangeHours.TimeZone, fillForward, extendedMarketHours,
-                    isInternalFeed, isCustomData,
-                    isFilteredSubscription: isFilteredSubscription, tickType: tickType,
+                select new SubscriptionDataConfig(
```

Comment:
I don't know about you, but this formatting makes it _soo_ much easier to actually read and see all of the parameters. Thanks!